$NetBSD: patch-aa,v 1.7 2011/07/05 04:28:34 dholland Exp $

Don't disable deprecated functions; more stuff has been deprecated
since this package was shipped. Don't use -ansi -pedantic either.

Configure for pkgsrc.

--- Makefile.orig	2005-02-28 15:00:56.000000000 +0000
+++ Makefile
@@ -2,12 +2,11 @@ PROG := gfract
 
 CC := g++
 
-DEFS := -Wall -ansi -pedantic -D_GNU_SOURCE -DGDK_DISABLE_DEPRECATED \
-	-DGTK_DISABLE_DEPRECATED
+DEFS := -Wall -D_GNU_SOURCE
 
 DEFS := $(shell pkg-config --cflags gtk+-2.0) $(DEFS)
 
-CXXFLAGS := $(DEFS) -O2 -fomit-frame-pointer -ffast-math
+CXXFLAGS += $(DEFS)
 #CXXFLAGS := $(DEFS) -g
 
 LDFLAGS := -lpng -lz
@@ -19,11 +18,11 @@ HEADERS := $(wildcard *.h *.xpm)
 
 $(PROG): $(OBJS)
 	@echo Linking $(PROG)
-	@$(CC) $(CXXFLAGS) $(OBJS) $(LDFLAGS) -o $(PROG)
+	@$(CXX) $(CXXFLAGS) $(OBJS) $(LDFLAGS) -o $(PROG)
 
 %.o: %.cpp $(HEADERS)
 	@echo Compiling $<
-	@$(CC) $(CXXFLAGS) -c $< -o $@
+	@$(CXX) $(CXXFLAGS) -c $< -o $@
 
 clean:
 	rm -f $(PROG) $(OBJS)
